src/BloomBrowserUI/bookEdit/toolbox/bookSettings/bookSettings.ts

```diff
@@ -22,7 +22,7 @@ $(document).ready(() => {
 export function handleBookSettingCheckboxClick(clickedButton: any) {
     // read our controls and send the model back to c#
     // enhance: this is just dirt-poor serialization of checkboxes for now
-    var inputs = $(".bookSettings :input");
+    var inputs = $("#bookSettings :input");
```

Comment:
At some point this got changed to an id instead of a class, and this code didn't get updated.